javascripts/social.js

```diff
@@ -1,14 +0,0 @@
-/* 
```

Comment:
Don't we need to make some changes to the gulp script to not add these files to the concatenation?